docs/microservices.json

```
@@ -1990,6 +1990,72 @@
       "apis": [],
       "version": null
     },
+    {
+      "id": "ia-case-documents-api",
+      "name": "IA case documents API",
+      "group": "IA",
+      "type": "api",
+      "description": "",
+      "repository": "https://github.com/hmcts/ia-case-documents-api",
+      "spec": "https://hmcts.github.io/reform-api-docs/specs/ia-case-documents-api.json",
+      "dependencies": [
+          {
+              "id": "ccd-data-store-api",
+              "hard": true,
+              "apis": []
+          },
+          {
+              "id": "rpe-service-auth-provider",
+              "hard": true,
+              "apis": []
+          },
+          {
+              "id": "cmc-pdf-service",
+              "hard": true,
+              "apis": []
+          },
+          {
+              "id": "idam",
```

i think this should be idam-api
```suggestion
              "id": "idam-api",
```

---

Corrected to "idam-api" for ia-case-api, ia-case-documents-api and ia-case-notifications-api 
as this is providing links to microservices and "idam" isn’t a microservice.